app/notification.go

```diff
@@ -140,33 +140,9 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 			delete(mentionedUserIds, post.UserId)
 		}
 
-		if len(m.OtherPotentialMentions) > 0 && !post.IsSystemMessage() {
-			if users, err := a.Srv.Store.User().GetProfilesByUsernames(m.OtherPotentialMentions, &model.ViewUsersRestrictions{Teams: []string{team.Id}}); err == nil {
-				channelMentions := model.UserSlice(users).FilterByActive(true)
-
-				var outOfChannelMentions model.UserSlice
-				var outOfGroupsMentions model.UserSlice
-
-				if channel.IsGroupConstrained() {
-					nonMemberIDs, err := a.FilterNonGroupChannelMembers(channelMentions.IDs(), channel)
-					if err != nil {
-						return nil, err
-					}
-
-					outOfChannelMentions = channelMentions.FilterWithoutID(nonMemberIDs)
-					outOfGroupsMentions = channelMentions.FilterByID(nonMemberIDs)
-				} else {
-					outOfChannelMentions = channelMentions
-				}
-				outOfChannelMentions = outOfChannelMentions.FilterWithoutBots()
-
-				if channel.Type != model.CHANNEL_GROUP {
-					a.Srv.Go(func() {
-						a.sendOutOfChannelMentions(sender, post, outOfChannelMentions, outOfGroupsMentions)
-					})
-				}
-			}
-		}
+		go func() {
+			_, _ = a.sendOutOfChannelMentions(sender, post, channel, m.OtherPotentialMentions)
```

Comment:
```suggestion
			a.sendOutOfChannelMentions(sender, post, channel, m.OtherPotentialMentions)
```

I'm surprised the formatter didn't break on this

Reply:
I actually did this so that anyone who reads this knows that we're ignoring the returned error on purpose. I can remove it if you'd like though.

Reply:
0/5 but how about a comment to make it more clear?

Reply:
I'm just going to add a log statement to print any error that occurs. I wasn't worried about it initially, but that seems like a better idea in case this does cause issues in the future.